mmv1/third_party/terraform/tests/resource_bigtable_instance_test.go

```diff
@@ -185,6 +185,131 @@ func TestAccBigtableInstance_kms(t *testing.T) {
 	})
 }
 
+func TestAccBigtableInstance_createWithAutoscalingAndUpdate(t *testing.T) {
+	// bigtable instance does not use the shared HTTP client, this test creates an instance
+	skipIfVcr(t)
+	t.Parallel()
+
+	instanceName := fmt.Sprintf("tf-test-%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckBigtableInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				// Create Autoscaling config with 2 nodes.
+				Config: testAccBigtableInstance_autoscalingCluster(instanceName, 2, 5, 70),
+				Check: resource.ComposeTestCheckFunc(resource.TestCheckResourceAttr("google_bigtable_instance.instance",
+						"cluster.0.num_nodes", "2"),),
```

Comment:
What are we checking here? only the allocated nodes?

Reply:
Yes, since that might not be set in the config, I want to explicitly check it.